Glossary of Terms.md

```diff
@@ -6,9 +6,9 @@ A brief list of terms as used in the [API Blueprint](http://apiblueprint.org) co
 
 <a name="def-action"></a>
 ### Action
-An **HTTP transaction** (a request-response transaction).
+One or more [**HTTP Transactions**](#def-transaction) within a [Resource](#def-resource) with a common [HTTP Request Method](#def-method).
 
-Actions are specified by an [HTTP request method](#def-method) within a [resource](#def-resource).
+[Requests](#def-request) and [Responses](#def-response) within an Action may be ordered into implicit groups where each such group represents one [**HTTP Transaction Example**](#def-transaction-example).
```

Comment:
Again see my comment above. Implicit is implied it is not something you can do explicitly.